modules/rgbd/test/test_tsdf.cpp

```diff
@@ -0,0 +1,515 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this module's directory
```

Comment:
Which part of test is a subject of separate license?

Reply:
Done